nptl/nptl_free_tcb.c

```diff
@@ -27,14 +27,17 @@ __nptl_free_tcb (struct pthread *pd)
   if ((atomic_fetch_or_relaxed (&pd->cancelhandling, TERMINATED_BITMASK)
       & TERMINATED_BITMASK) == 0)
     {
-      /* Free TPP data.  */
-      if (pd->tpp != NULL)
-        {
-          struct priority_protection_data *tpp = pd->tpp;
+      // Qianxi Edit: Thread Local storage is not supported yet
+      // so we do not need to free it
+      
+      // /* Free TPP data.  */
```

Comment:
does this not work if we uncomment it?

Reply:
This would crash for some reason. So I have to comment these out